packages/contracts-bedrock/test/SystemConfig.t.sol

```diff
@@ -98,7 +98,7 @@ contract SystemConfig_Initialize_Test is SystemConfig_Initializer {
         vm.expectEmit(true, true, true, true, address(sysConf));
         emit ConfigUpdate(0, SystemConfig.UpdateType.BATCHER, abi.encode(batcherHash));
         vm.expectEmit(true, true, true, true, address(sysConf));
-        emit ConfigUpdate(0, SystemConfig.UpdateType.GAS_CONFIG, abi.encode(overhead, scalar));
+        emit ConfigUpdate(0, SystemConfig.UpdateType.GAS_CONFIG, abi.encode(gasConfig));
```

Comment:
If this test passes, then the encoding is equivalent to referencing the properties, i thought it would be the same but its one of those things that could create a bug in some unexpected way

Reply:
Yeah, agree. I didn't really want to have a stand alone test sitting around that verifies it though. I can unwrap this event too lmk.